scripts/assemble_visualization_webpage.py

```diff
@@ -143,7 +151,7 @@ def get_primers_data(full_path, consensus):
     if primer_locations:
         primers_map[description] = arrange_gff_data(primer_locations)
     else:
-        print("No primer was mapped from ", path, ".")
+        print("No primer was mapped from <%s>." % full_path)
```

Comment:
Are we targeting Python>=3.6?
If yes, we should use f-strings.

Reply:
I think ShoRAH has also requirement about python 3.6 too.
And we basically target bioconda with V-pipe and conda has python 3.8 at least.